db/seeds.rb

```diff
@@ -12,24 +12,7 @@
              password_confirmation: 'password',
              admin: false
 
-Field.create! name: 'Curb cut',
-              category: 'Accessible',
-              field_type: :choice,
-              choices: ["Within 20'", 'No curb cut', 'No curb'],
-              description: 'Determine the border between the road and the bus stop.',
-              rank: 1
-
-Field.create! name: 'Solar lighting',
-              category: 'Technology',
-              field_type: :boolean,
-              description: 'Determine if this stop has solar lighting.',
-              rank: 1
-
-Field.create! name: 'Completed',
-              category: 'Info',
-              field_type: :boolean,
-              description: 'Have all attributes been checked?',
-              rank: 1
+# Recommended to create fields with `rails bus_stop_fields:create`.
```

Comment:
Ok

Reply:
Seems like we wouldn't want a different set of fields in dev vs. prod, similar to how right now we don't have a different set of columns on the bus stop table.

Reply:
Got i